benchmarks/Bio/multisite2.jmd

```diff
@@ -85,13 +91,13 @@ test_sol  = TestSolution(sol)
 abstols = 1.0 ./ 10.0 .^ (6:10)
 reltols = 1.0 ./ 10.0 .^ (6:10);
 setups = [
-          Dict(:alg=>lsoda()),
-          Dict(:alg=>CVODE_BDF(linear_solver=:GMRES)),
-          Dict(:alg=>TRBDF2(linsolve=KrylovJL_GMRES())),
-          Dict(:alg=>QNDF(linsolve=KrylovJL_GMRES())),
-          Dict(:alg=>FBDF(linsolve=KrylovJL_GMRES())),
-          Dict(:alg=>KenCarp4(linsolve=KrylovJL_GMRES())),
-          Dict(:alg=>Rosenbrock23(linsolve=KrylovJL_GMRES())),
+          Dict(:alg=>lsoda()), 
+          Dict(:alg=>CVODE_BDF(linear_solver=:GMRES)), 
+          Dict(:alg=>QNDF()), 
+          Dict(:alg=>Tsit5()), 
+          Dict(:alg=>BS5()), 
+          Dict(:alg=>Vern6()), 
+          Dict(:alg=>Vern8())

```

Comment:
This was the stiff ODE method benchmark, why was this changed?

Reply:
I thought we said we'd use explicit methods for the two smaller models in Zulip? I  can move multisite2  back to implicit solvers only, but I think the explicit ones perform slightly better.

Reply:
There was one benchmark on stiff methods, and one benchmark on the non-stiff methods. This PR updates to make both of them on the non-stiff methods, with the same methods. I don't understand why it would need two different plots of almost the the same thing?

Reply:
Wait, I am probably missing something. So there is the multistate and multisite2 models  (with the second one being larger). For both explicit methods are better, so I  benchmark these (admittedly the same set, but could differentiate a bit). In each case, I also threw in the best stiff solver just to show how one of these performs.

I think it makes sense to benchmark on both multistate and multisite2.  I could include more/different stiff/non-stiff solvers in each case. I could also make one plot with stiff and one with non-stiff solvers in each case.

Reply:
it's already setup as a plot of stiff then non-stiff, so I am wondering why that was changed.

Reply:
Sorry, I will go through an check this when I get to this to figure out exactly what I did and why.

Reply:
Not sure if this was something that wasn't there in the version I worked on it, and now it got messed up in the merge. Or I just missed it when I thought I'd use the updated solver set. Re-adding explicit solvers for all benchmarks.